compiler/ilgen/IlBuilder.cpp

```diff
@@ -516,16 +516,17 @@ IlBuilder::appendBlock(TR::Block *newBlock, bool addEdge)
    _currentBlock = newBlock;
    }
 
-
 void
-IlBuilder::AppendBuilder(TR::IlBuilder *builder)
+IlBuilder::AppendBuilder(TR::IlBuilder *builder, TR::Block *blockThrowsException)
    {
-   ILB_REPLAY("%s->AppendBuilder(%s);", REPLAY_BUILDER(this), REPLAY_BUILDER(builder));
+   ILB_REPLAY("%s->AppendBuilder(%s) isExceptionBuilder: %d;", REPLAY_BUILDER(this), REPLAY_BUILDER(builder), isExceptionBuilder);
```

Comment:
The string output by ILB_REPLAY is not a log message. It is a line of C++ code that captures the call that has been made. So you need to update the string and the argument to capture the arguments to AppendBuilder.